docs/coding/php_guidelines.md

```diff
@@ -0,0 +1,173 @@
+### PSR-12 Guidelines for PHP
+
+PSR-12 builds upon PSR-1 and PSR-2, setting comprehensive coding standards for PHP. Following these guidelines ensures clean, readable, and maintainable code. Here’s a guide to implementing PSR-12 in PHP projects:
+
+**1. Basic Coding Standards:**
```

Comment:
these should be h2 tag

Reply:
I have made the changes. Can you please review it again?